azavea/echo-locator
# Feature/kak/neighborhood details#28

## Overview

 - Add neighborhood details page
 - Implement saving neighborhoods to favorites
 - Implement switching between full list and favorites in sidebar
 - Show modes/travel time/origin summary on details
 - Link to Google Maps search for neighborhood's zip code from details page


### Demo

Detail summary, with list of unique modes and reference to the current destination type (day care).
![image](https://user-images.githubusercontent.com/960264/55016865-ff300700-4fc5-11e9-9e03-3cd8fd7f970f.png)

List, with new summary heading, toggled to favorites:
![image](https://user-images.githubusercontent.com/960264/55016942-238be380-4fc6-11e9-91cc-dacd410f2c75.png)


### Notes

Finalizing the neighborhood details and list card information is still pending changes to the source data. This PR is to enable styling work to progress without the new data.


## Testing Instructions

 * Clicking a neighborhood point on the map should go to its detail view in the sidebar
 * Clicking the header for a neighborhood list card should also go to the detail view
 * Should be able to favorite/unfavorite neighborhoods from list or detail views
 * Toggling between favorites ("saved") and all neighborhood lists should work as expected
 * Links to Google Maps should go to the neighborhood postal code area
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case, and expected output


Connects #28 
Closes #33
